### DISCUSSION TITLE
Polish language

### DISCUSSION THREAD

**Discussion by macgyverkam** [pakerwreah/Calendr#184] [Localization]
Your application is great! Can I somehow translate it officially into Polish?

**Comment by pakerwreah** [comment]
Yes, please take this PR as an example:
- #164

If you are not familiar with Xcode, just copy [this file](https://github.com/pakerwreah/Calendr/blob/master/Calendr/Assets/en.lproj/Localizable.strings), edit it with your translation and attach it here.
